src/main/java/org/fundacionjala/coding/marcoslara/spinwords/SpinWords.java

```diff
@@ -1,32 +0,0 @@
-package org.fundacionjala.coding.marcoslara.spinwords;
```

Comment:
Please apply the workaround
1. git reset HEAD~3 (to revert the commits)
2. git commit again without removing those files
3. git push --force